packages/editor/src/components/post-title/index.js

```diff
@@ -186,6 +173,13 @@ const applyWithDispatch = withDispatch( ( dispatch ) => {
 		onUndo: undo,
 		onRedo: redo,
 		clearSelectedBlock,
+		onSelect() {
+			togglePostTitleSelection( true );
+			clearSelectedBlock();
+		},
+		onUnselect() {
```

Comment:
I didn't know es6 allowed us to declare functions this way. Thanks for the TIL! 😎